services/app-api/utils/reports/reports.ts

```diff
@@ -15,32 +16,43 @@ export async function copyFieldDataFromSource(
   state: string | undefined,
   copyFieldDataSourceId: any,
   formTemplate: any,
-  validatedFieldData: AnyObject
+  validatedFieldData: AnyObject,
+  reportType: ReportType
 ) {
   const sourceFieldData = (await s3Lib.get({
     Bucket: reportBucket,
     Key: getFieldDataKey(state as State, copyFieldDataSourceId),
   })) as AnyObject;
 
-  if (sourceFieldData) {
+  if (sourceFieldData && reportType === ReportType.MCPAR) {
```

Comment:
Nit: When 90% of the body of a function is wrapped in an `if`, I tend to prefer early returns for the negative cases. For example, we could check the report type before even touching S3:
```ts
if (reportType !== ReportType.MCPAR) {
  return validatedFieldData;
}
```
The same style of "I don't have any work to do, so I'm bailing" early return could be used again to de-nest the rest of the function:
```ts
const sourceFieldData = ...
if (!sourceFieldData) {
  return validatedFieldData;
}
const possibleFields = ...
```